jerry-port/default/default-date.c

```diff
@@ -13,6 +13,9 @@
  * limitations under the License.
  */
 
+#ifdef HAVE_TM_GMTOFF
+#include <time.h>
+#endif
 #ifdef __GNUC__
 #include <sys/time.h>
 #endif
```

Comment:
This patch does not affect this code path, but I would be great to unify the closing `#endif` comments as well.